src/api/spec/features/webui/maintained_projects_spec.rb

```diff
@@ -1,6 +1,6 @@
 require 'browser_helper'
 
-RSpec.feature 'Bootstrap_MaintainedProjects', type: :feature, js: true, vcr: true do
+RSpec.feature 'MaintainedProjects', type: :feature, js: true, vcr: true do
```

Comment:
As there is no generated cassettes for MaintainedProjects, I would remove from this line the `vcr: true`.